src/pages/groupComparison/GroupComparisonMutationMapper.tsx

```diff
@@ -7,32 +7,107 @@ import {
 import {
     AxisScale,
+    DataFilterType,
     groupDataByGroupFilters,
+    onFilterOptionSelect,
     ProteinImpactTypeBadgeSelector,
 } from 'react-mutation-mapper';
 import _ from 'lodash';
-import { ComparisonGroup } from './GroupComparisonUtils';
+import {
+    ComparisonGroup,
+    getProteinChangeToMutationRowData,
+    SIGNIFICANT_QVALUE_THRESHOLD,
+} from './GroupComparisonUtils';
 import DriverAnnotationProteinImpactTypeBadgeSelector from 'shared/components/mutationMapper/DriverAnnotationProteinImpactTypeBadgeSelector';
 import { IAnnotationFilterSettings } from 'shared/alterationFiltering/AnnotationFilteringSettings';
 import SettingsMenuButton from 'shared/components/driverAnnotations/SettingsMenuButton';
-import { ProteinImpactWithoutVusMutationType } from 'cbioportal-frontend-commons';
 import styles from './styles.module.scss';
 import { LegendColorCodes } from 'shared/components/mutationMapper/LegendColorCodes';
+import { ProteinImpactWithoutVusMutationType } from 'cbioportal-frontend-commons';
+import MutationTable, {
+    ExtendedMutationTableColumnType,
+} from 'shared/components/mutationTable/MutationTable';
+import GroupComparisonMutationTable from './GroupComparisonMutationTable';
+import MutationMapperDataStore, {
+    PROTEIN_CHANGE_FILTER_ID,
+} from 'shared/components/mutationMapper/MutationMapperDataStore';
+import { extractColumnNames } from 'shared/components/mutationMapper/MutationMapperUtils';
+import autobind from 'autobind-decorator';
+import { Sample } from 'cbioportal-ts-api-client';
+import { FisherExactTwoSidedTestLabel } from './FisherExactTwoSidedTestLabel';
+import ComplexKeyMap from 'shared/lib/complexKeyDataStructures/ComplexKeyMap';
+import { CheckedSelect, Option } from 'cbioportal-frontend-commons';
+import { SHOW_ALL_PAGE_SIZE as PAGINATION_SHOW_ALL } from '../../shared/components/paginationControls/PaginationControls';
 
 interface IGroupComparisonMutationMapperProps extends IMutationMapperProps {
     onInit?: (mutationMapper: GroupComparisonMutationMapper) => void;
     axisMode?: AxisScale;
     onScaleToggle?: (selectedScale: AxisScale) => void;
     groups: ComparisonGroup[];
     annotationFilterSettings: IAnnotationFilterSettings;
+    groupToProfiledPatients: {
+        [groupUid: string]: string[];
+    };
+    sampleSet: ComplexKeyMap<Sample>;
+    profiledPatientCounts: number[];
 }
 
 @observer
 export default class GroupComparisonMutationMapper extends MutationMapper<
     IGroupComparisonMutationMapperProps
 > {
+    @observable.ref _selectedGroupsForEnrichedInFilter: string[];
+    @observable significanceFilterEnabled: boolean = false;
+
     constructor(props: IGroupComparisonMutationMapperProps) {
         super(props);
+        makeObservable(this);
+
+        this._selectedGroupsForEnrichedInFilter = this.props.groups.map(
+            group => group.nameWithOrdinal
+        );
+    }
+
+    @computed get paginationStatusText(): string {
```

Comment:
This is partially duplicate of https://github.com/cBioPortal/cbioportal-frontend/blob/9ce8d4e6c9216e86f7b8d00d653ccecc74f2faef/src/shared/components/lazyMobXTable/LazyMobXTable.tsx#L594

Not sure how much effort it would take to make this a utility function to share/reuse.